script/test

```diff
@@ -152,7 +152,8 @@ do_check()
 {
     (cd "${OTBR_TOP_BUILDDIR}" \
         && ninja && sudo ninja install \
-        && CTEST_OUTPUT_ON_FAILURE=1 ninja test)
+        && CTEST_OUTPUT_ON_FAILURE=1 ctest -LE sudo \
```

Comment:
document this line is for excluding the `sudo` test cases.

Reply:
We cannot add a comment after `\`...  I don't find any good ways to document this line. I updated the comment of the next line. Does it work for you?